xml/security-security_features.xml

```diff
@@ -6,15 +6,43 @@
  <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&productnamereg;: Security Features Overview</title>
+ <title>&productnamereg;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security Planning</title>
+  <para>
+   &productnamereg; is a complex system which has many options which
+   affect the security of the system and the workloads running within it. Careful planning
+   is required to ensure the correct options and features are chosen for your particular
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html">Security Boundaries and Threats</link>
+   to understand how OpenStack defines security domains. Ensure that the network layout
+   you choose for your deployment provides adequate network separation for your specific
+   requirements. The other parts of the
+   <link xlink:href="https://docs.openstack.org/security-guide/">OpenStack Security Guide</link>
+   are also informative.
```

Comment:
This sentence strikes me funny. Presumably everything in the guide is informative. 
Suggest: We suggest careful study of the full (link)OpenStack Security Guide(/link).

Reply:
addressed in https://github.com/SUSE-Cloud/doc-cloud/pull/950